setup.py

```diff
@@ -22,20 +22,14 @@
     license="Apache 2.0",
     packages=["arrow"],
     zip_safe=False,
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
-    install_requires=[
-        "python-dateutil>=2.7.0",
```

Comment:
Is there a reason for removing the version limit for dateutil here? We need 2.7.0 for most of our DST improvements.

Reply:
nope, I think it was a lapse on my part during the merge of updating to master. I will add it back.